packages/terminal/src/browser/terminal-widget-impl.ts

```diff
@@ -374,6 +374,10 @@ export class TerminalWidgetImpl extends TerminalWidget implements StatefulWidget
         }
     }
 
+    // Device status code emitted by Xterm.js
+    // Check: https://github.com/xtermjs/xterm.js/blob/master/src/InputHandler.ts#L1107
```

Comment:
Detail, but links pointing to a file on master are likely to break. Specific commits, or even release branches are better. E.g. https://github.com/xtermjs/xterm.js/blob/release/3.14/src/InputHandler.ts#L1055-L1082